### PR TITLE
merge with fb upstream cra 4.0

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

**Review by calebmshafer — APPROVED** [review]
LGTM, thanks for picking this up @aruniverse!

Let's get this merged in and we can start testing it out with 4.0.0-beta.0 package.  I'm interested to try out the new FAST_REFRESH in the iModel.js repo and see how it works.

Also, we'll want to make sure we do a real merge commit instead of a squash commit.

**Comment by calebmshafer** [comment]
@aruniverse I'll merge upstream master into our master branch now. Plus open the PR for updating the docs to match what I outlined in #8.

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "@bentley/react-scripts",
-  "version": "3.4.6",
-  "description": "iModel.js configuration and scripts for Create React App.",
+  "version": "4.0.0-beta.0",
+  "description": "Configuration and scripts for Create React App.",
```
[the remark this answers is not in the document]
Comment (aruniverse) [reply packages/react-scripts/package.json:4]: no, bad merge, ill update. I'll also change the version to use the dev pre-release tag

**Comment by calebmshafer** [comment]
Updated the `master` branch to match upstream.

**Comment by aruniverse** [comment]
Abandoning in favor of #24

**Pull request closed without merging**